crates/holochain/src/conductor/conductor.rs

```diff
@@ -1276,9 +1275,10 @@ mod network_impls {
 
 /// Methods related to app installation and management
 mod app_impls {
-
     use super::*;
+
     impl Conductor {
+        #[cfg(feature = "test_utils")]
         pub(crate) async fn install_app_legacy(
```

Comment:
IIRC this is only used in sweetest, maybe it can be that feature instead of `test_utils`?

Reply:
I could do, the sweetest feature is currently just there for consumers so they can request `sweetest` and we are then free to change what that does as we make changes. Everything in the code is still referring `test_utils`.

What I wanted to do with this and have stayed away from because it's a much bigger change is trying to separate fixt/sweetest/test_utils within the holochain repository. They are interdependent in a few places and although I think I can separate them I would then need to move the functional tests that use sweetest out of the `holochain` crate to avoid circular dependencies. I think that's probably not what we want to do because in places it looks like its useful for the test tools to mock private code etc. So for now, I think making this easier to consume is a useful step

Reply:
I'm glad you're thinking along those lines. I hope we do a major reorg of the crates some day, which would include breaking up `holochain` into smaller chunks and layers, including pulling out sweettest as its own crate -- which would include moving the tests as you say -- so that devs can use sweettest without importing all of `holochain`. So it's going to be a question of when it's appropriate to refactor, and how much refactoring we can stand, and what the benefits will be. Like an electrical charge building up, will it all be released at once in a mighty bolt (major refactor), or will there be some smaller sparks over time to reduce the charge buildup more gradually (more PRs like this)?

If we plan any smaller refactors, let's also think of the larger refactors we might want so we're heading in the right direction. What you say feels like a move in the right direction.